sys/storage/Makefile

```diff
@@ -0,0 +1,7 @@
+MODULE = turpial_firmware_storage
```

Comment:
`storage` is fine, other names were that way to not cause collisions with riot modules.